### PR TITLE
Tmz 1.6.0 complex numbers

### PR DESCRIPTION
**Pull request by t-m-z**
converts between cartesian and polar coordinates

### CODE REVIEW

File: assets/i18n/en.json [hunk 1]
```diff
@@ -974,7 +974,17 @@
   "common_unit_volume_smi_name": "US Smidgen",
   "common_unit_volume_pn_name": "US Pinch",
   "common_unit_volume_ds_name": "US Dash",
-  "coords_selection_title": "Coordinate Calculation",
```
Comment (Mike-3) [inline assets/i18n/en.json:977]: not remove coords_selection_title

**Review by Mike-3 — COMMENTED** [review]
character input posibile. Please change, if possible.
Possibly, for example, 1/3 to be accepted as a valid input (use math_expressions)

**Review by Mike-3 — APPROVED** [review]

**Comment by S-Man42** [comment]
Same issue as on quadratic: Please add some space between the inputs :)

Another thing I saw: Why can I enter any character? I guess there should be an input mask for only [\-0-9,.]. Don't you think?

**Comment by t-m-z** [comment]
> Same issue as on quadratic: Please add some space between the inputs :)

Layout issue is solved

> Another thing I saw: Why can I enter any character? I guess there should be an input mask for only [-0-9,.]. Don't you think?

Hm - I used your DoubleTextInputFormatter - maybe there is a bug in the regex?

**Comment by t-m-z** [comment]
PR done

**Comment by S-Man42** [comment]
I'll check the input issue...

**Pull request merged**